packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -150,6 +150,7 @@ interface DashboardOptions<M extends Meta, B extends Body>
   theme?: 'auto' | 'dark' | 'light'
   trigger?: string
   width?: string | number
+  autoOpen?: boolean | 'imageEditor' | 'metaEditor'
```

Comment:
The user facing type does not allow a `boolean`.

```suggestion
  autoOpen?: 'imageEditor' | 'metaEditor'
```